include/swift/Basic/LangOptions.h

```diff
@@ -371,6 +371,24 @@ namespace swift {
       return EffectiveLanguageVersion.isVersionAtLeast(major, minor);
     }
 
+    // The following Apple platforms ship an Objective-C runtime supporting
+    // the class metadata update callback mechanism:
+    //
+    // - macOS 10.14.4
+    // - iOS 12.2
+    // - tvOS 12.2
+    // - watchOS 5.2
+    bool doesPlatformSupportObjCMetadataUpdateCallback() const;
+
+    // The following Apple platforms ship an Objective-C runtime supporting
+    // the objc_getClass() hook:
+    //
+    // - macOS 10.14.4
+    // - iOS 12.2
+    // - tvOS 12.2
+    // - watchOS 5.2
+    bool doesPlatformSupportObjCGetClassHook() const;
```

Comment:
Nitpick: these should be "target" or "deployment target" or "minimum deployment target", not "platform". "Platform" doesn't include a version number.